pkg/server/server.go

```diff
@@ -225,24 +218,9 @@ func (s *Server) GetHealth(_ context.Context, request *tetragon.GetHealthStatusR
 	return health.GetHealth()
 }
 
-func (s *Server) ListSensors(ctx context.Context, _ *tetragon.ListSensorsRequest) (*tetragon.ListSensorsResponse, error) {
+func (s *Server) ListSensors(_ context.Context, _ *tetragon.ListSensorsRequest) (*tetragon.ListSensorsResponse, error) {
 	logger.GetLogger().Debug("Received a ListSensors request")
-	list, err := s.observer.ListSensors(ctx)
-	if err != nil {
-		logger.GetLogger().WithError(err).Warn("Server ListSensors request failed")
-		return nil, err
-	}
-
-	sensors := make([]*tetragon.SensorStatus, 0, len(*list))
-	for _, s := range *list {
-		sensors = append(sensors, &tetragon.SensorStatus{
-			Name:       s.Name,
-			Enabled:    s.Enabled,
-			Collection: s.Collection,
-		})
-	}
-
-	return &tetragon.ListSensorsResponse{Sensors: sensors}, nil
+	return nil, fmt.Errorf("ListSensors is deprecated")
```

Comment:
why don't we remove all this code?

Reply:
You mean the gRPC calls?
I tried to, but the `buf`linter said it's not OK because it breaks backwards compatibility: https://buf.build/docs/breaking/rules/#service_no_delete.

🤷🏼